lib/travis/build/script/smalltalk.rb

```diff
@@ -76,6 +76,22 @@ def script
 
         private
 
+          def smalltalk_ci_repo
+            if config.key?(:smalltalk_edge) and config[:smalltalk_edge].key?(:source)
+              return config[:smalltalk_edge][:source]
```

Comment:
I wonder if this one-liner for `smalltalk_ci_repo ` is better (not sure if this is something people prefer in Ruby):
```ruby
return config.fetch(:smalltalk_edge, {}).fetch(:source, "hpi-swa/smalltalkCI")
```